controllers/automation/policyautomation_controller.go

```diff
@@ -199,6 +201,121 @@ func (r *PolicyAutomationReconciler) Reconcile(
 			} else {
 				log.Info("All clusters are compliant. Doing nothing.")
 			}
+		} else if policyAutomation.Spec.Mode == "everyEvent" {
+			log := log.WithValues("mode", "everyEvent")
+			targetList := common.FindNonCompliantClustersForPolicy(policy)
+			eventTime := time.Now().UTC().Format(time.RFC3339)
+			if len(targetList) > 0 {
+				delayAfterRunSecondsStr := policyAutomation.Spec.DelayAfterRunSeconds
+				delayAfterRunSeconds, err := strconv.Atoi(delayAfterRunSecondsStr)
```

Comment:
This can be removed after changing to the type of `DelayAfterRunSeconds` to `uint`.